stickers.yml

```diff
@@ -20377,3 +20377,26 @@ d62864e473972014274b69df859cc2ff:
     - Telegram
   nsfw: false
   original: false
+
+08d662207baa720bb2a5385950f52777:
+  key: aee175897ba161ad9b8bb9702d505f6cd27107c06589f8dd3345505fd2f5469c
+  source: LINE
+  tags:
+    - Hatsune
+    - Miku
+    - 初音ミク
+    - Vocaloid
+    - Magical Mirai
+    - cute
+    - blue hair
+    - anime
+    - kawaii
+    - food
+    - chibi
+    - LINE
+   
+  nsfw: false
+  original: false
+  https://signalstickers.com/pack/08d662207baa720bb2a5385950f52777?key=aee175897ba161ad9b8bb9702d505f6cd27107c06589f8dd3345505fd2f5469c
```

Comment:
This line should be removed. This is the preview URL that we usually ask you to include in the Pull Request description, but it should not be in the YAML.